doc/DeveloperGuide.md

```diff
@@ -88,6 +88,34 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename a tag
+2. AddressBook requests user to enter name of tag to rename and new name for the tag.
```

Comment:
see if you can split this to multiple steps?
same for step 4.
this is also OK, but splitting would make it more granular

Reply:
Alright! thanks for the advice!